store/tikv/gcworker/gc_worker.go

```diff
@@ -612,6 +671,83 @@ func doGCForOneRegion(bo *tikv.Backoffer, store tikv.Storage, safePoint uint64,
 	return nil, nil
 }
 
+func genNextGCTask(store tikv.Storage, bo *tikv.Backoffer, safePoint uint64, key kv.Key) (*gcTask, error) {
+	loc, err := store.GetRegionCache().LocateKey(bo, key)
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+
+	task := &gcTask{
+		startKey:  key,
+		endKey:    loc.EndKey,
+		safePoint: safePoint,
+	}
+	return task, nil
+}
+
+func doGCParallel(ctx goctx.Context, store tikv.Storage, safePoint uint64, identifier string, concurrency int) error {
```

Comment:
Make this function as a method of `GCWorker` is better.
We can remove `RunGCJob` as it is only used by a bench program.

Reply:
ok